### ISSUE TITLE
Add endpoint to get rule violations of ORT run

### ISSUE THREAD

**Issue by mnonnenmacher** [eclipse-apoapsis/ort-server#795]
Add an endpoint to get the rule violations of an ORT run.

The location of the endpoint should be "/runs/{id}/rule-violations".

**Comment by Etsija** [comment]
`{id}` means the global id, not `ortRunIndex`?

**Comment by mnonnenmacher** [comment]
> `{id}` means the global id, not `ortRunIndex`?

Yes, all endpoints at "/runs/" use the global id as the repository id is unknown.

**Comment by sschuberth** [comment]
This was done in https://github.com/eclipse-apoapsis/ort-server/pull/1095.
